app/src/components/common/outcomes/index.tsx

```diff
@@ -109,10 +109,11 @@ interface Props {
   onChange: (newOutcomes: Outcome[]) => any
   errorMessages?: string[]
   totalProbabilities: number
+  disabled: boolean
 }
 
 const Outcomes = (props: Props) => {
-  const { outcomes, totalProbabilities, errorMessages } = props
+  const { outcomes, totalProbabilities, disabled = false, errorMessages } = props
```

Comment:
The prop is mandatory so the default value doesn't seem necessary.

Reply:
Done!